<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoleChat</class>
 <widget class="QMainWindow" name="MoleChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 116, 117);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(6, 76, 162);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="left_frame">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(6, 76, 162);
}

/*VERTICAL SCROLLBAR*/
QScrollBar:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	width: 14px;
	margin: 15px 0 15px 0;
	border-radius: 0px;
}

/*HANDLE BAR*/
QScrollBar::handle:vertical{
	background-color: rgb(0, 0, 166);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::handle:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*BTN TOP SCROLLBAR*/
QScrollBar::sub-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::sub-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*BTN BOTTOM SCROLLBAR*/
QScrollBar::add-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::add-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*RESET ARROW*/
QScrollBar::up-arrow::vertical, QScrollBar::down-arrow:vertical{
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="user_info_area" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(4, 47, 102);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Logo/ava_mole.gif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="pixmap">
             <pixmap>../../../../Downloads/ava_mole.gif</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(229, 229, 229)</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Meepo Geomancer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="user_other_area" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>46</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>46</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(4, 47, 102);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="control_button_1">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	font-size: 20px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="user_other_area2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(4, 47, 102);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="chat_list_area">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>244</width>
            <height>557</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="chat_list">
             <property name="styleSheet">
              <string notr="true">QListWidget{
	font: 22px;
	border: none;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="right_frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(6, 76, 162);
}

QScrollBar:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	width: 14px;
	margin: 15px 0 15px 0;
	border-radius: 0px;
}


QScrollBar::handle:vertical{
	background-color: rgb(0, 0, 166);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::handle:vertical:pressed{
	background-color: rgb(159, 155, 204);
}


QScrollBar::sub-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::sub-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}


QScrollBar::add-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::add-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}


QScrollBar::up-arrow::vertical, QScrollBar::down-arrow:vertical{
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="chat_tittle_area" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(4, 47, 102);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="chat_ava">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../../../../Downloads/1024px-Close-up_of_mole.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="chat_name">
            <property name="styleSheet">
             <string notr="true">color: rgb(229, 229, 229)</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Welcome, mr.Mole!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chat_button1">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	font-size: 20px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chat_button2">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	font-size: 20px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chat_button_3">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 15px;
	border-top-right-radius: 15px;
	border-bottom-left-radius: 15px;
	border-bottom-right-radius: 15px;
	font-size: 20px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="chat_area">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>550</width>
            <height>586</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QWidget" name="chat_message_area" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(6, 58, 126);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>9</number>
          </property>
          <item alignment="Qt::AlignBottom">
           <widget class="QPushButton" name="gif_button">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
	font-size: 30px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="message_input">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit{
	background-color: rgb(142, 167, 255);
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
	padding: 9px 15px 9px;
}

/*HANDLE BAR*/
QScrollBar::handle:vertical{
	background-color: rgb(0, 0, 166);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::handle:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*BTN TOP SCROLLBAR*/
QScrollBar::sub-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::sub-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*BTN BOTTOM SCROLLBAR*/
QScrollBar::add-line:vertical{
	border: none;
	background-color: rgb(0, 0, 102);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover{
	background-color: rgb(122, 115, 199);
}

QScrollBar::add-line:vertical:pressed{
	background-color: rgb(159, 155, 204);
}

/*RESET ARROW*/
QScrollBar::up-arrow::vertical, QScrollBar::down-arrow:vertical{
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background: none;
}</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="placeholderText">
             <string>Enter the message</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QPushButton" name="send_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*BUTTON Forma*/
QPushButton {
	background-color: rgb(88, 99, 255);
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
	font-size: 30px;
}

QPushButton:hover {
	background-color: rgb(122, 115, 199);
}

QPushButton:pressed {
	background-color: rgb(159, 155, 204);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../Downloads/logo.qrc"/>
 </resources>
 <connections/>
</ui>
